bark/python_wrapper/models/behavior.cpp

```diff
@@ -387,7 +389,50 @@ void python_behavior(py::module m) {
             return new BehaviorNotStarted(
                 PythonToParams(t[0].cast<py::tuple>()));
           }));
+  
+  py::class_<BehaviorSafety, BehaviorModel,
+             shared_ptr<BehaviorSafety>>(m, "BehaviorSafety")
+    .def(py::init<const bark::commons::ParamsPtr&>())
+    .def("SetBehaviorModel", &BehaviorSafety::SetBehaviorModel)
+    .def("__repr__",
+      [](const BehaviorSafety& b) {
+        return "bark.behavior.BehaviorSafety";
+      })
+    .def(py::pickle(
+      [](const BehaviorSafety& b) {
+        return py::make_tuple(
+          ParamsToPython(b.GetParams()));
+      },
+      [](py::tuple t) {
+        if (t.size() != 1)
+          throw std::runtime_error("Invalid behavior model state!");
+        /* Create a new C++ instance */
+        return new BehaviorSafety(
+          PythonToParams(t[0].cast<py::tuple>()));
+      }));
 
+  py::class_<BehaviorRSSConformant, BehaviorModel,
+             shared_ptr<BehaviorRSSConformant>>(m, "BehaviorRSSConformant")
+    .def(py::init<const bark::commons::ParamsPtr&>())
+    .def("SetNominalBehaviorModel", &BehaviorRSSConformant::SetNominalBehaviorModel)
+    .def("SetSafetyBehaviorModel", &BehaviorRSSConformant::SetSafetyBehaviorModel)
+    .def("__repr__",
+      [](const BehaviorRSSConformant& b) {
+        return "bark.behavior.BehaviorRSSConformant";
+      })
+    .def(py::pickle(
+      [](const BehaviorRSSConformant& b) {
+        return py::make_tuple(
+          ParamsToPython(b.GetParams()));
```

Comment:
Does not serialize safety and nominal behavior models -> segfaults in benchmarking

Reply:
But the models will be set in the constructor to the default values?

Reply:
? how to resolve?

Reply:
Implement getters for nominal and safety models and use them in the serialization:
py::make_tuple(
          ParamsToPython(b.GetParams()), ParamsToPython(b.GetNominalModel()->GetParams(), ParamsToPython(b.GetSafetyModel()->GetParams());

For deserialization, build recreate the nomal and safety models and use the setters to set them for RSSConformantModel